admin/templates/osf_groups/osf_groups_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% load node_extras %}
+
+{% load static %}
+{% block title %}
+    <title>OSF Groups</title>
+{% endblock title %}
+{% block content %}
+<h2>List of Groups</h2>
+{% include "util/pagination.html" with items=page status=status %}
+<table class="table table-striped table-hover table-responsive">
+    <thead>
+        <tr>
+            <td>Name</td>
+            <td>Date Created</td>
+            <td>Date Modified</td>
+            <td>Creator</td>
+        </tr>
+    </thead>
+    <tbody>
+        {% for group in groups %}
+        <tr>
+            <td>
+                <a href="{{ group | reverse_osf_group }}">{{ group.name }}</a>
```

Comment:
Same here, if you're using the group serializer for the GroupsList view, then you'll need to update a few references in this template.
One example:
```
 <a href="{{ group.named_id | reverse_osf_group }}">{{ group.name }}</a>
```